content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -104,18 +96,20 @@ istio-pilot-58c65f74bc-2f5xn             1/1       Running   0          1m
 
 * For option 2, uninstall using Helm:
 
+> Uninstalling this chart does not delete Istio's registered CRDs. Istio, by design, expects
+> CRDs to leak into the Kubernetes environment. As CRDs contain all runtime configuration
+> data in custom resources the Istio designers consider better to explicitly delete this
```

Comment:
`data in custom resource` -> `data for Istio configuraiton`.  This sentence is a little clunky.